### ISSUE TITLE
Provide support for passing "=" as a string 

### ISSUE THREAD

**Issue by sivudu47** [StackStorm/st2#6160]
**alias yaml**
`---
name: "launch_quasar"
action_ref: "quasar.quasar1"
description: "launch a quasar execution"
formats:
  - display: "*<command>* *<payload>*"
    representation:
      - "{{ command }} {{ payload }}"
result:
  format: |
    ```{{ execution.result.result }}```
`
**action yaml**

name: quasar1
description: Action that takes an input parameter
runner_type: 'python-script'
entry_point: 'quasar1.py'
enabled: true
parameters:
  command:
    type: string
    description: 'Input parameter'
    required: true
  payload:
    type: string
    description: 'Input parameter'
    required: true
  user:
    type: "string"
    description: "Slack user who triggered the action"
    required: false
    default: "{{action_context.api_user}}"

**Representation used:** "{{ command }} {{ payload }}"
**Parameter defintion**:
  payload:
    type: string
    description: 'Input parameter'
    required: true

We have an usecase where we need to pass a string with "=" in it for payload for some reason stackstorm is not allowing me to do that, if i pass such value it is not taking it as a seperate string and causing multiple issues

Issue is not oberved if we are putting ":" instead of "=" and also putting a space after "=" solves the issue

Have tried different things (using jinja template replace option and replaced "=" with "= " but i am getting internal server error. 
if i am able to pass the yaml validation, i can have my own validation in action py file but the issue the execution is not even going to the python file.

basically this should be accepted by stackstorm  **quasar create cluster_name=weekly_nats**

<img width="372" alt="Screenshot 2024-03-04 at 11 35 28 AM" src="https://github.com/StackStorm/st2/assets/41072130/ae316000-c84a-4493-976e-e2e6f5b38fbc">
